apprise/plugins/__init__.py

```diff
@@ -61,6 +61,9 @@
 ]
 
 
+PLUGINS_WITH_MODERNIZED_TESTS = ["NotifyEmby"]
```

Comment:
Of course, this will only be temporary until all test cases have been adjusted.

Reply:
It will not go in at all, because #687 will resolve all issues on this matter. Putting this PR back into draft mode now.